eva_sub_cli/validators/docker_validator.py

```diff
@@ -7,26 +7,23 @@
 
 from ebi_eva_common_pyutils.logger import logging_config
 
-from eva_sub_cli.validators.validator import Validator, VALIDATION_OUTPUT_DIR
+from eva_sub_cli.validators.validator import Validator
 
 logger = logging_config.get_logger(__name__)
 
 container_image = 'ebivariation/eva-sub-cli'
-container_tag = 'v0.0.1.dev6'
+container_tag = 'v0.0.1.dev7'
 container_validation_dir = '/opt/vcf_validation'
 container_validation_output_dir = 'vcf_validation_output'
 
 
 class DockerValidator(Validator):
 
-    def __init__(self, mapping_file, output_dir, metadata_json=None,
+    def __init__(self, mapping_file, submission_dir, metadata_json=None,
                  metadata_xlsx=None, container_name=None, docker_path='docker', submission_config=None):
-        # validator write to the validation output directory
-        # If the submission_config is not set it will also be written to the VALIDATION_OUTPUT_DIR
-        super().__init__(mapping_file, os.path.join(output_dir, VALIDATION_OUTPUT_DIR),
```

Comment:
I don't think there was any particular reason why the `DockerValidator` output directory had to be different from the `NativeValidator` one, so I changed it to be consistent between the two...